### ISSUE TITLE
searching for ingredient

### ISSUE THREAD

**Issue by nathaliavalli** [cis3296f24/project-01-Super-Chef#46]
Everytime I ran the ingredient egg I got different list of recipe

**Comment by HarpinderSingh1999** [comment]
I don't think this one is in our control, it might be the API we use. But we can expand the pagination aspect we were talking about to like mitigate this.
